charts/ocis/templates/appregistry/deployment.yaml

```diff
@@ -99,5 +99,5 @@ spec:
           emptyDir: {}
         - name: configs
           configMap:
-            name: {{ .appName }}-registry-config
+            name: {{ .appName }}-config
```

Comment:
We should add `registry` to the configMap instead as this was the way it was before, see https://github.com/owncloud/ocis-charts/blob/c07dc3062a0a7dcc299309aa1006b67820a9c249/charts/ocis/templates/app-registry/config.yaml#L5

Reply:
No this was before the templating. So nowadays it needs to be `{{ .appName }}-registry-config` in the `config.yaml`

Reply:
Ah right. The renaming added that in the `appName` already. Then all is fine.